config/default.yml

```diff
@@ -3610,6 +3610,11 @@ Style/RedundantPercentQ:
   Enabled: true
   VersionAdded: '0.76'
 
+Style/RedundantRegexpEscape:
+  Description: 'Checks for redundant escapes in Regexps.'
+  Enabled: true
+  VersionAdded: '0.85'
```

Comment:
I'm not sure if the next version will be `0.85`, or if it will be `1.0`. @bbatsov knows.

Reply:
Seems it will be 0.85. Too much development happening recently. I want to give @marcandre time to finish the things he has in progress.